manifest.yaml

```diff
@@ -29,7 +29,7 @@ repos:
   - rhel-8-server-ose
 
 # We include hours/minutes to avoid version number reuse
-automatic-version-prefix: "48.83.<date:%Y%m%d%H%M>"
+automatic-version-prefix: "48.84.<date:%Y%m%d%H%M>"
```

Comment:
What we should probably do along with this is explicitly pin the expected `redhat-release-coreos` RPM NVR, because it needs to be in sync with this change.